exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
I guess I should ask: what are the various sourcesets used for? This is one of the areas in which the Kotlin and Java tracks diverge.